odk1-src/aggregate-vm.rst

```diff
@@ -1,4 +1,65 @@
-ODK Aggregate Virtual Machine 
-=================================
+.. spelling::
 
-The `ODK Aggregate VM <https://gumroad.com/l/odk-aggregate-vm>`_ is a fully-configured install of Aggregate that you can run on any computer. It requires very little setup, works well without Internet connectivity, and gives you complete control over your data collection campaign.
+  readme
+  VirtualBox
+
+ODK Aggregate Virtual Machine
+=============================
+
+This document provides instructions on setting up ODK Aggregate using the `ODK Aggregate VM OVA file <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ and `VirtualBox <https://www.virtualbox.org>`_.
```

Comment:
The first mention of a main product (Aggregate) should normally be a link to the intro page:

```
:doc:`aggregate-intro`
```

This will insert the text as "ODK Aggregate".
Subsequent references to the product name don't need to include "ODK", just "Aggregate.

(The same goes for Collect, Briefcase, etc.)